src/main/java/com/librarymanagement/DatabaseConn.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.librarymanagement;
+import java.sql.*;
+import java.util.logging.Level;
+import java.sql.Connection;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.Executor;
+import java.util.logging.Logger;
+/**
+ *
+ * @author praneeth
+ */
+public class DatabaseConn {
+    
+    static final String JDBC_DRIVER="com.mysql.jdbc.Driver";
+	static final String DB_URL="jdbc:mysql://localhost/test";
+	static final String USER="SRUTHI";
+	static final String PASS="password";
+	Connection conn=null;
+   private static DatabaseConn sqlconn;
+   
+    private static DatabaseConn getinstance(){
+        if(sqlconn==null){
+            sqlconn=new DatabaseConn();
+        }
+               
+        
+        return sqlconn;
+    }
+    public void setconnection(){
+       
+        
+        try {
+          
+            Class.forName(JDBC_DRIVER);
+            System.out.println("Connecting to selected database");
```

Comment:
Please remove println statements and logging.
Please refrain from using println statement